client/src/pages/about-us/index.js

```diff
@@ -117,9 +119,12 @@ const AboutUs = () => {
           We build and use strategies to drive digital growth and expand customer reach so you can focus on growing your business!
         </IntroductionDescription>
       </IntroductionContainer>
-
+      <BrowseContainer>
+        Browse Our Past Work
+      </BrowseContainer>
+      <BorderContainer></BorderContainer>
```

Comment:
<img width="957" alt="codeReviewss" src="https://user-images.githubusercontent.com/91857946/191125308-b5b65b91-d9e8-4040-85a5-dd5eff316ad5.png">

I saw this code's project through front end, and I was able to see this red-to-blue gradient border. I was wondering if there's specific width for this border specified, or is it supposed to be good with the current width? Just pointing this out since it looked awkward on the about-us front-end page.